lib/games_collection.rb

```diff
@@ -12,4 +12,59 @@ def generate_objects_from_csv(csv_games_path)
     end
     objects
   end
+
+  # Helper method designed to be reusable; consider moving to a module
+  def every(attribute, collection)
+    collection.map do |element|
+      element.send(attribute)
+    end
+  end
+
+  # Helper method designed to be reusable; consider moving to a module
+  def every_unique(attribute, collection)
+    every(attribute, collection).uniq
+  end
```

Comment:
can't we just put .uniq on the end of the enumerable above? then we would get the same support method and one less method taking up space? or is this ment to just show a list on uniq. attributes and not an iteration?

Reply:
We could, yep; I decided to make them separate because I foresaw having a use for the one without .uniq (in future methods). I'd be okay with combining them if we end up never using the other one by itself.